README.md

```diff
@@ -115,7 +148,30 @@ An example of extra specs json would look like this:
+    }
+```
+
+*NOTE*: The `extra_context` spec adds a map of key/value pairs that are expanded in the `runner_install_template`.
+The `runner_install_template` allows us to completely override the script that installs and starts the runner. In the example above, I have added a copy of the current template from `garm-provider-common`, with the adition of one line:
+
+```bash
+echo "I am the override script: {{ .ExtraContext.Person }} - {{ .ExtraContext.Who }}" | sudo tee /garm-override.txt
```

Comment:
Replace this with something a bit more friently/meaningful for the example. Mine was terrible.

Reply:
Updated it to do a curl to `github.com` and paste the output into a file